Tests/StreamChatUITests/TestImages/XCTestCase+TestImages.swift

```diff
@@ -16,6 +16,14 @@ extension XCTestCase {
             getImage(withName: "yoda")
         }()
         
+        static let commandGiphy: (url: URL, image: UIImage) = {
+            getImage(withName: "Giphy", fileExtension: "png")
+        }()
+        
+        static let commandFallback: (url: URL, image: UIImage) = {
+            getImage(withName: "Fallback", fileExtension: "png")
+        }()
+        
```

Comment:
It's confusing when you use real images :) It must be clear to everyone these are TEST images. Maybe we need more images than Yoda and Vader ?